transbank/onepay/transaction.py

```diff
@@ -108,6 +108,7 @@ def create(cls, shopping_cart: ShoppingCart, channel = Channel.WEB, external_uni
 
         external_unique_number_req = external_unique_number or int(datetime.now().timestamp() * 1000)
         options = Options.build(options)
+        print(options.shared_secret)
```

Comment:
Don't leave `print`s on commited code (especially not for sensitve information!)

Reply:
print removed